### PR TITLE
Theme & Introduction Redesign

### PR DESCRIPTION
**Pull request by BboyAkers**

### CODE REVIEW

**Comment by BboyAkers** [comment]
@Gudahtt @danfinlay @rekmarks Would love to get y'alls honest feedback on this docs redesign 🙂. Please pull this branch and take a look! I'll be creating more PRs for different parts of the docs but this is just a start! I'm more than happy to revert or heavily modify changes if needed 😄

File: docs/.vuepress/styles/index.styl [hunk 1]
```diff
@@ -39,10 +39,34 @@ pre.vue-container
...
 .primaryBtn {
   background-color: #037dd6;
 }
 
 .greenBtn {
   background-color: #4caf50;
 }
+
+.navbar, .links {
+  background-color: #D7C1B3 !important
```
Comment (Gudahtt) [inline docs/.vuepress/styles/index.styl:55]: Wow, bold choice in colour :sweat_smile: Seems a bit...plain, and off-brand for us maybe? :man_shrugging: Maybe it's just me.

**Review by Gudahtt — COMMENTED** [review]
I like the direction you're going in with the introduction! It looks a lot friendlier.

Where did these images come from? Did you produce them yourself? :astonished:

File: docs/.vuepress/styles/index.styl (hunk 1 above)
Comment (BboyAkers) [reply docs/.vuepress/styles/index.styl:55]: I got it from ya'lls logo and I think design palette. Could someone from ya'lls design team confirm please? 🙂

**Comment by BboyAkers** [comment]
> I like the direction you're going in with the introduction! It looks a lot friendlier.
> 
> Where did these images come from? Did you produce them yourself? 😲

I got it from https://undraw.co/illustrations 🙂 they're free! I made some modifications to them in photoshop to add the metamask logo and stuff!

License:
https://undraw.co/license

**Comment by rekmarks** [comment]
It does look a lot friendlier! The orange links are a nice touch. I'd be curious to see what the design team has to say. cc: @MetaMask/design  

<details>
<summary>Click here for screenshot</summary>

</details>

File: docs/.vuepress/styles/index.styl (hunk 1 above)
Comment (rekmarks) [reply docs/.vuepress/styles/index.styl:55]: This is the one thing I don't like! I'd prefer that we just go with white unless we find something more suitable. Maybe @cjeria has some ideas.

Comment (rekmarks) [reply docs/.vuepress/styles/index.styl:55]: Design agrees that white header is better for now :)

**Review by rekmarks — CHANGES_REQUESTED** [review]
Design requests that the links are changed to this color:

`#037DD6`

**Comment by BboyAkers** [comment]
Updated! 🙂

**Comment by rekmarks** [comment]
Is there any way to center the lone images under the section headers? E.g., this one:
image

**Comment by BboyAkers** [comment]
@rekmarks fixed it! and centered the images 🙂 Sorry for the wait!

**Comment by rekmarks** [comment]
@BboyAkers sorry for the delay. 🙇‍♂️ 

I asked our designers for feedback, and we agreed that it would be better to remove the images you added. We really appreciate the time and effort you put into them, but we decided against using them because:

1. Any written document should maximize its signal-to-noise ratio, and we judge that the images won't help developers understand MetaMask better, or find the information they're looking for faster.
2. If we decide to add more visual elements to our documentation, that'll be a longer process that involves the design team.

The remaining changes you made still look good, and as always we appreciate your efforts to improve our documentation!

**Comment by BboyAkers** [comment]
No worries 🙂! I'll remove the images and push those changes today haha

**Pull request closed without merging**